provider/resources.go

```diff
@@ -0,0 +1,142 @@
+
+		// The GitHub Org for the provider - defaults to `terraform-providers`. Note that this
+		// should match the TF provider module's require directive, not any replace directives.
+		Config: map[string]*tfbridge.SchemaInfo{
+			// Add any required configuration here, or remove the example below if
+			// no additional points are required.
+			// "region": {
+			// 	Type: tfbridge.MakeType("region", "Region"),
+			// 	Default: &tfbridge.DefaultInfo{
+			// 		EnvVars: []string{"AWS_REGION", "AWS_DEFAULT_REGION"},
+			// 	},
+			// },
+		},
+		PreConfigureCallback: preConfigureCallback,
+		Resources: map[string]*tfbridge.ResourceInfo{
+			// Map each resource in the Terraform provider to a Pulumi type. Two examples
+			// are below - the single line form is the common case. The multi-line form is
+			// needed only if you wish to override types or other default options.
+			//
+			// "aws_iam_role": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "IamRole")}
+			//
+			// "aws_acm_certificate": {
+			// 	Tok: tfbridge.MakeResource(mainPkg, mainMod, "Certificate"),
+			// 	Fields: map[string]*tfbridge.SchemaInfo{
+			// 		"tags": {Type: tfbridge.MakeType(mainPkg, "Tags")},
+			// 	},
+			// },
+			"frontegg_allowed_origin":      {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AllowedOrigin")},
+			"frontegg_redirect_uri":        {Tok: tfbridge.MakeResource(mainPkg, mainMod, "RedirectUri")},
+			"frontegg_permission":          {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Permission")},
+			"frontegg_permission_category": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "PermissionCategory")},
+			"frontegg_role":                {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Role")},
+			"frontegg_tenant":              {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Tenant")},
+			"frontegg_user":                {Tok: tfbridge.MakeResource(mainPkg, mainMod, "User")},
+			"frontegg_webhook":             {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Webhook")},
+			"frontegg_workspace":           {Tok: tfbridge.MakeResource(mainPkg, mainMod, "Workspace")},
+		},
+		DataSources: map[string]*tfbridge.DataSourceInfo{
+			// Map each resource in the Terraform provider to a Pulumi function. An example
+			// is below.
+			// "aws_ami": {Tok: tfbridge.MakeDataSource(mainPkg, mainMod, "getAmi")},
+			"frontegg_permission": {Tok: tfbridge.MakeDataSource(mainPkg, mainMod, "getPermission")},
+		},
+		Python: &tfbridge.PythonInfo{
+			// List any Python dependencies and their version ranges
+			Requires: map[string]string{
+				"pulumi": ">=3.0.0,<4.0.0",
+			},
+		},
+	}
+
+	// These are new API's that you may opt to use to automatically compute resource tokens,
+	// and apply auto aliasing for full backwards compatibility.
+	// For more information, please reference: https://pkg.go.dev/github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge#ProviderInfo.ComputeTokens
+	// prov.MustComputeTokens(tokens.SingleModule("frontegg_", mainMod,
```

Comment:
any reason you commented this out?

Reply:
I honestly don't know.. a lot of this is auto-genearted, we didn't use this in our provider prior to this update and the pulumi-materialize provider doesn't use this.
In general I don't want to remove a lot of auto-generated options if they could be used down the line, but I'm happy to here if it's preferred.